app/controllers/course/assessment/submissions_controller.rb

```diff
@@ -24,6 +24,14 @@ def edit
   end
 
   def update
+    if params[:submit_answer]
```

Comment:
@lowjoel Other than doing this, do you have suggestions ? I can't find a way to make a form able to submit to different controllers.